.github/SECURITY.md

```diff
@@ -0,0 +1,19 @@
+# Security Policy
+
+As careful and community-oriented maintainers, we strive to keep our code safe to use for everyone. 
+For this purpose, we highly value your input and will acknowledge your contributions as best we can.
+
+## Supported Versions
+
+Unfortunately, our team size does not permit us to support multiple versions at once. 
+We generally develop our `main` branch and so to get the version of our code that contains all currently available security updates, please [install our code from source](https://docs.messageix.org/en/latest/install-adv.html#install-from-github) using the `main` branch. 
+Please reach out to us if you encounter any issues in doing so.
+
+If this is not an option, please ensure you are using the [latest release of our code](https://github.com/iiasa/message_ix/releases) to profit from as many security updates as possible.
+We strive to publish new releases at least twice a year, so there should never be too many security updates pending.
```

Comment:
“`message_ix` is generally released twice a year, but may make additional releases on a shorter time line if necessary to mitigate critical vulnerabilities.”